tests/config_file_meta/CONUS/nwm_ana/hydro.namelist

```diff
@@ -18,24 +18,24 @@
     hydrotbl_f = './NWM/DOMAIN/hydro2dtbl_FullRouting_NWMv2.0.nc'
     igrid = 1
     io_config_outputs = 0
-    io_form_outputs = 2
+    io_form_outputs = 4
     land_spatial_meta_flnm = './NWM/DOMAIN/GEOGRID_LDASOUT_Spatial_Metadata_1km_NWMv2.0.nc'
     lsmout_domain = 1
     nsoil = 4
     order_to_write = 1
     out_dt = 60
     outlake = 1
-    output_channelbucket_influx = 2
+    output_channelbucket_influx = 0
```

Comment:
Is there a reason to turn these off?

Reply:
Hawaii now appears to be inconsistent w/ CONUS

Reply:
... I guess we are doing the sneakier way here but not above. I will look at HI.  This is a good point.